package.json

```diff
@@ -20,6 +20,6 @@
     "eslint-plugin-react": "^7.33.2",
     "lerna": "^7.4.2",
     "npm-run-all": "^4.1.5",
-    "prettier": "^3.1.0"
+    "prettier": "3.1.0"
```

Comment:
Why pinned version here?

Reply:
Prettier wants you to pin a specific version since any version changes in prettier could result in lots of file changes. Note the `--save-exact` here:

https://prettier.io/docs/en/install